src/App.js

```diff
@@ -5,6 +5,9 @@ import Home from "./pages/home/Home";
 import Login from "./pages/login/Login";
 import Signup from "./pages/signup/Signup";
 import VideoListing from "./pages/video listing/VideoListing";
+import Playlist from './pages/playlist/Playlist'
+import SpecificPlaylist from "./pages/playlist/specific playlist/SpecificPlaylist";
```

Comment:
You can have index.js file in your pages folder from which you can export all the pages.